tests/remote_obj_test.go

```diff
@@ -113,3 +114,85 @@ func TestConsoleLogParse(t *testing.T) {
 		})
 	}
 }
+
+func TestEvalRemoteObjectParse(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name string
+		eval string
+		want any
+	}{
+		{
+			name: "number", eval: "1", want: 1,
+		},
+		{
+			name: "string", eval: `"some string"`, want: "some string",
+		},
+		{
+			name: "bool", eval: "true", want: true,
+		},
+		{
+			name: "empty_array", eval: "[]", want: []any{},
+		},
+		{
+			name: "empty_object", eval: "{}", want: goja.Undefined(),
+		},
+		{
+			name: "filled_object", eval: `{return {foo:"bar"};}`, want: map[string]any{"foo": "bar"},
+		},
+		{
+			name: "filled_array", eval: `{return ["foo","bar"];}`, want: []interface{}{0: "foo", 1: "bar"},
+		},
+		{
+			name: "filled_array", eval: `() => true`, want: `function()`,
+		},
+		{
+			name: "empty", eval: "", want: "",
+		},
+		{
+			name: "null", eval: "null", want: "null",
+		},
+		{
+			name: "undefined", eval: "undefined", want: goja.Undefined(),
+		},
+		{
+			name: "bigint", eval: `BigInt("2")`, want: 2,
+		},
+		{
+			name: "unwrapped_bigint", eval: "3n", want: 3,
+		},
+		{
+			name: "float", eval: "3.14", want: 3.14,
+		},
+		{
+			name: "scientific_notation", eval: "123e-5", want: 0.00123,
+		},
+		// {
```

Comment:
WDYT about? It's good for tracking them. For instance, my text editor shows me all TODOs in a list.

```suggestion
		// TODO:
		// {
```